framework/web/AssetManager.php

```diff
@@ -482,7 +482,13 @@ protected function publishFile($src)
 
         if ($this->linkAssets) {
             if (!is_file($dstFile)) {
-                symlink($src, $dstFile);
+                try { // fix #6226 symlinking multi threaded
+                    symlink($src, $dstFile);
+                } catch (\Exception $e) {
+                    if (!is_file($dstFile)) {
```

Comment:
What if there was a file with the same name you tried creating symlink with?

Reply:
this already is a very rare usecase. It highly likely that the symlink is already correct because it did not exist a fraction of time ago https://github.com/yiisoft/yii2/pull/14905/files/54ac1820734bc85dd10b31492c16751b7582b94e#diff-8b967e883b8604768a6f7251d1f1285cR484

Therefore, if an error occurs and it is because a file already exist, it is because of a collision of the same symlink action (happening in a multi threading env)

Reply:
OK. Makes sense.